dsc/tests/dsc_config_get.tests.ps1

```diff
@@ -56,6 +56,31 @@ Describe 'dsc config get tests' {
         $LASTEXITCODE | Should -Be 0
     }
 
+    It 'json progress for config subcommand' {
+        $config_yaml = @"
+            `$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+            resources:
+            - name: Echo
+              type: Microsoft.DSC.Debug/Echo
+              properties:
+                output: hello
+"@
+        $config_yaml | dsc config get --output-format json -f - 2> $TestDrive/ErrorStream.txt
+        $LASTEXITCODE | Should -Be 0
+        $lines = Get-Content $TestDrive/ErrorStream.txt
+        $ProgressMessagesFound = $False
+        foreach ($line in $lines) {
+            if ($line.StartsWith('{')) {
```

Comment:
This check shouldn't be needed, just have the trace format be JSON and if any text comes through, that's a bug

Reply:
updated